at_client/src/main/java/org/atsign/client/util/ConvertingUtils.java

```diff
@@ -0,0 +1,60 @@
+import java.io.*;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+// Main class
+
+public class ConvertingUtils {
+
+     // Converting file object to byte array
+    public static byte[] convert(File file)
+        throws IOException
+    {
+  
+        //creating an instance of file
+        FileInputStream filel = new FileInputStream(file);
+  
+        //creating a byte array with same length as file
+        byte[] byteArr = new byte[(int)file.length()];
+  
+       //Reading file content 
+        filel.read(byteArr);
+  
+        //Closing the file
+        filel.close();
+  
+        // Returning byte array
+        return byteArr;
+    }
+
+  
+  //Converting byte array to file
+    public static File convert(byte[] bytes)
+    {
+        //paste yout destination path here
+        String FILEPATH="";  
```

Comment:
Hey @yashjain1974 thanks for your work =)

Could you receive the file path as an argument instead of hard coding it in?